cpp/dolfinx/fem/Expression.h

```diff
@@ -114,8 +115,13 @@ class Expression
     // Create data structures used in evaluation
     std::vector<double> coordinate_dofs(3 * num_dofs_g);
 
+    const int num_all_argument_dofs
```

Comment:
If the caller needs this to interpret the output of the function then it should go in a public member function?